packages/gafl-webapp-service/src/pages/licence-details/licence-start-time/__tests__/route.spec.js

```diff
@@ -137,3 +144,79 @@ describe('licenceStartTimeRoute > onPostAuth', () => {
     })
   })
 })
+
+describe('getData', () => {
+  const getData = pageRoute.mock.calls[1][4]
+  const mockTransactionCacheGet = jest.fn()
+  const mockRequest = {
+    cache: () => ({
+      helpers: {
+        transaction: {
+          getCurrentPermission: mockTransactionCacheGet
+        }
+      }
+    }),
+    locale: 'en-gb'
+  }
+
+  it('startDateStr is result of moment().format()', async () => {
+    const formatSymbol = Symbol('formatResult')
+    moment.mockImplementation(
+      getMomentMockImpl({
+        format: () => formatSymbol
+      })
+    )
+    mockTransactionCacheGet.mockImplementationOnce(() => ({ licenceStartDate: '2020-01-01' }))
+    const { startDateStr } = await getData({
+      cache: () => ({
+        helpers: {
+          transaction: {
+            getCurrentPermission: mockTransactionCacheGet
+          }
+        }
+      }),
+      locale: 'anywhere'
+    })
+    expect(startDateStr).toEqual(formatSymbol)
+  })
+
+  it.each([['2020-07-06'], ['2020-01-01'], ['2022-07-09']])('permission licence start date is passed to moment', async licenceStartDate => {
+    mockTransactionCacheGet.mockImplementationOnce(() => ({ licenceStartDate }))
+    moment.mockImplementation(getMomentMockImpl())
+    await getData(mockRequest)
+    expect(moment).toHaveBeenCalledWith(licenceStartDate, 'YYYY-MM-DD', 'en-gb')
+  })
```

Comment:
This could be simplified a bit - I think you could make licenceStartDate a `Symbol` and then you wouldn't need different test cases, just one. Also, you don't care what the second and third arguments are, so do:
```
expect(moment).toHaveBeenCalledWith(licenceStartDate, expect.any(String), expect.any(String))
```